### DISCUSSION TITLE
Visualizing with VMD

### DISCUSSION THREAD

**Discussion by ghost** [mir-group/nequip#158] [Q&A] [answered]
Hello,
Thank you for sharing the project. I was wondering if you could tell me how I can use the result of toluene simulation to visualize in VMD?

**Comment by Linux-cpp-lisp** [comment, marked as answer]
I'm assuming you're referring to the Colab tutorial, where we use LAMMPS. In that context, `pair_nequip` is like any other potential in LAMMPS, and things like dumping and visualization are similar to any other use of LAMMPS. Please see their docs (https://lammps.org/viz.html) and the various tutorials available online for LAMMPS, running and dumping LAMMPS simulations, and visualizing LAMMPS results.

**Comment by ghost** [comment]


**Reply by Linux-cpp-lisp** [reply]
Would you please make a new thread/discussion for this question so we can keep this as a fairly searchable resource? Thanks
